### ISSUE TITLE
AptioMemoryFix Deprecated

### ISSUE THREAD

**Issue by averycblack** [Errrneist/Hackintosh-Thinkpad-X1-Extreme#60]
AptioMemoryFix is deprecated and got wrapped into OpenCore and FWRuntimeServices.efi.
Using [OCQuirks](https://github.com/ReddestDream/OcQuirks), it's possible to use FWRuntimeServices.efi with OcQuirks instead of AptioMemoryFix.

**Comment by ishtiaq156** [comment]
Yes. I can confirm that OCquirks+FwRuntimeServices efis instead of Aptio* works fine. This resolved my slide value error I keep getting it every boot.

**Comment by wu-hongjun** [comment]
> AptioMemoryFix is deprecated and got wrapped into OpenCore and FWRuntimeServices.efi.
> Using [OCQuirks](https://github.com/ReddestDream/OcQuirks), it's possible to use FWRuntimeServices.efi with OcQuirks instead of AptioMemoryFix.

Currently, I am not familiar with OpenCore and therefore might not deprecate AptioMemoryfix quite yet cuz I am busy with other developments and my current EFI works perfectly (unless this entire whole thing becomes an issue in the future). You are more than welcome to discuss this topic though...

**Comment by averycblack** [comment]
Makes sense.
OcQuirks.efi and FWRuntimeServices are just drop in replacements for AptioMemoryFix. A recent PR for OcQuirks also allowed the FwRuntimeservices to be configured from a plist if the default values contained within don't work, but the default values usually are fine.
